src/css/popup.css

```diff
@@ -362,7 +404,7 @@ onboarding-panel p {
   right: 30px;
 }
 
-.maxAliasesPanel img {
```

Comment:
Does changing this specificity affect other images on this panel?

Reply:
I changed it to the specific class name so that it wouldn't affect the `img` object in the new upgrade banner.

Reply:
I changed it to the specific class name so that it wouldn't affect the `img` object in the new upgrade banner.